src/modules/user/user.service.ts

```diff
@@ -37,14 +44,22 @@ export class UserService {
       length: 10,
       numbers: true,
     });
-    console.log(
-      '-------- The generated password is -------- ',
-      theUser.email,
-      password,
-    );
+
     const salt = await genSalt(10);
     theUser.password = await hash(password, salt);
-    return plainToClass(UserResponseDto, await theUser.save());
+
+    await theUser.save()
+
+    const fullName = `${theUser.firstname} ${theUser.lastname}`;
+
+    await this._sendGrid.send({
+      to: theUser.email,
+      from: new ConfigService().get(ConfigEnum.BUSINESS_MAIL),
+      subject: "Welcome to Fast Order",
+      html: emailTemplate(fullName, theUser.email, password),
```

Comment:
Aquí, creo que sería mejor un object como parametro, para no tener que enviar tantos

Reply:
i get it, i'm going to remove it